### PR TITLE
Class team

### PR DESCRIPTION
**Pull request by jiayinysj**

### CODE REVIEW

File: src/main/java/backend/Team.java [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+    public Team(String name, String manager, int budget, int pointsAllTime, int pointsThisSeason,
+                int winAllTime, int winThisSeason, Engine engine,
+                Aerodynamicist aerodynamicist, Mechanic mechanic, Strategist strategist) {
+        this.name = name;
+        this.manager = manager;
+        this.budget = budget;
+        this.pointsAllTime = pointsAllTime;
+        this.pointsThisSeason = pointsThisSeason;
+        this.winAllTime = winAllTime;
+        this.winThisSeason = winThisSeason;
```
Comment (Steptro) [inline src/main/java/backend/Team.java]: pointsAlltime, pointsThisSeason, winAllTime and winThisSeason can be removed for the constructor

File: src/main/java/backend/Team.java [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+import java.util.ArrayList;
+import java.util.List;
+
+public class Team {
+    private String name;
+    private String manager;
+    private int budget;
+    private int pointsAllTime;
+    private int pointsThisSeason;
+    private int winAllTime;
```
Comment (Steptro) [inline src/main/java/backend/Team.java]: Field should be called winsAlltime

File: src/main/java/backend/Team.java [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+    private int winThisSeason;
```
Comment (Steptro) [inline src/main/java/backend/Team.java]: Field should be called winsThisSeason

File: src/main/java/backend/Team.java (hunk 1 above)
Comment (jiayinysj) [inline src/main/java/backend/Team.java]: Should I remove these completely from this class? And wrap it in a new class?

File: src/main/java/backend/Team.java [hunk 4]
```diff
@@ -0,0 +1,318 @@
...
+    private Engine engine;
+    private Aerodynamicist aerodynamicist;
+    private Mechanic mechanic;
+    private Strategist strategist;
+    private List<Driver> drivers;
+
+    /**
+     * Create a new team.
+     * Create a new arraylist for drivers.
```
Comment (Steptro) [inline src/main/java/backend/Team.java]: Not necessar to add a comment for that. The reader can see that directly from the code

File: src/main/java/backend/Team.java (hunk 1 above)
Comment (Steptro) [inline src/main/java/backend/Team.java]: No only remove them in the constructor as parameters. For now just initialzie them with 0 in the constructor

**Comment by Steptro** [comment]
How can this be merged while some checks failed??

**Pull request merged**